src/frontend/lib/file-system.ts

```diff
@@ -1,3 +1,45 @@
+import * as FileSystem from 'expo-file-system';
+import * as DocumentPicker from 'expo-document-picker';
+
+// @ts-expect-error Only null when on web https://github.com/expo/expo/issues/5558
+export const DOCUMENT_DIRECTORY: string = FileSystem.documentDirectory;
+
 export function convertFileUriToPosixPath(fileUri: string) {
   return fileUri.replace(/^file:\/\//, '');
 }
+
+// TODO: Some overlap with selectFile() from lib/utils but fixes some usage limitations. Ideally use this for everything
```

Comment:
just noting this but if there's an interest in unifying the implementations, I'd rather address in a follow up